core/class.go

```diff
@@ -56,7 +57,24 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt
+	PythonicHints   json.RawMessage
+	CompilerVersion string
+	Hints           json.RawMessage
+	Prime           *big.Int
+	External        []CompiledEntryPoint
+	L1Handler       []CompiledEntryPoint
+	Constructor     []CompiledEntryPoint
+}
+
+type CompiledEntryPoint struct {
+	Offset   uint64
+	Builtins []string
```

Comment:
Do you think we should have a `Builtin` enum?

Reply:
I believe adding enum variants is a breaking change that should only happen across network versions.

But yeah, probably not worth making this an enum right now.